modules/login/php/requestaccount.class.inc

```diff
@@ -271,5 +271,46 @@ class RequestAccount extends \NDB_Form
 
         // Show success message even if email already exists for security reasons
         $this->tpl_data['success'] = true;
+
+        // Send an email to all users meeting the following criteria:
+        //     - Associated with the Site
+        //     - Associated with the Project
+        //     - Have the User Management permission
+        $query  = "SELECT DISTINCT u.Email,u.userID
+             FROM users u
+                JOIN user_psc_rel upr
+                    (ON u.ID = upr.UserID)
+               JOIN user_project_rel uprojr
+                   (ON u.ID = uprojr.UserID)
+            WHERE upr.CenterID=:center
+                AND uprojr.ProjectID=:project";
+        $result = $DB->pselect(
+            $query,
+            ['center' => $site, 'project' => $project]
+        );
+        foreach ($result as $row) {
+            $hasPermission = \User::factory($row['userID'])
+                ->hasAnyPermission(
+                    ['user_accounts', 'user_accounts_multisite']
+                );
+            if ($hasPermission === false) {
+                continue;
+            }
+
+            // Send email to users with User Management permissions.
+            $config    = \NDB_Factory::singleton()->config();
+            $studyName = $config->getSetting('title');
+
+            \Email::send(
```

Comment:
This is still missing a sandbox check

Reply:
unless I am misreading this, I believe the sandbox check is done line 276 (returns from the function and this code is not executed).

Reply:
nevermind, I guess this was resolved before John left. Will resolve this comment.